bin/scripts/export-data/getAllEventsSegmentsForApp.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Output list of app names, event keys, and segments.
+ * Server: countly
+ * Path: bin/scripts/export-data
+ * Command: node getallEventsSegmentsForApp.js
+ */
+
+var plugins = require("../../../plugins/pluginManager.js");
+
+plugins.dbConnection("countly").then(function(db) {
+    db.collection("events").find().toArray(function(err, events) {
+        db.collection("apps").find().toArray(function(err, names) {
+
+            if (err) {
```

Comment:
Here we need to check for both errors from events and app, so the names or arguments in those callbacks should be different and in the if you would check both with || operator

Reply:
I modified the code to reflect these changes! :)